src/app/modules/submission/submission.service.ts

```diff
@@ -251,17 +258,64 @@ export const copyPendingSubmissionToSubmissions = (
       return okAsync(submission)
     })
     .andThen((pendingSubmission) => {
-      const submission = new SubmissionModel(
-        omit(pendingSubmission, ['_id', 'created', 'lastModified']),
-      )
+      const submissionContent = omit(pendingSubmission, [
+        '_id',
+        'created',
+        'lastModified',
+      ])
+      const submission = new SubmissionModel({
+        // Explicitly copy over the pending submission's _id
+        _id: pendingSubmissionId,
+        ...submissionContent,
```

Comment:
```suggestion
        ...submissionContent,
        _id: pendingSubmissionId,
```
Since we want to make sure the `_id` for this part is `pendingSubmissionId`, right?

Reply:
makes sense, [431ec70](https://github.com/opengovsg/FormSG/pull/6088/commits/431ec7014bce214100600093b9f93d5e6f56801a)